### DISCUSSION TITLE
Number section only starting a certain level

### DISCUSSION THREAD

**Discussion by Pierre9344** [quarto-dev/quarto-cli#8203] [Q&A] [answered]
### Description

Hello,
I'm currently trying to use quarto to render a docx document with unnumbered section that contains sub-section that should be numbered.
My problem is that when I render the document, the sub-sections are named `0.1 name`, `0.2 name` and not `1 title`. Is there a way to start the numbering only after a certain level in the toc?

I thank you in advance for your answers

````
---
title: "test"
format: docx
toc: true
number-sections: true
---

# Section 1 (not numbered)  {.unnumbered}

## Quarto

Quarto enables you to weave together content and executable code into a finished document. To learn more about Quarto see <https://quarto.org>.

## Numbered section


# Section 2 (not numbered)  {.unnumbered}


## Running Code

When you click the **Render** button a document will be generated that includes both content and the output of embedded code. You can embed code like this:

```{r}
1 + 1
```

You can add options to executable code like this 

```{r}
#| echo: false
2 * 2
```
````
[rendered document](https://github.com/quarto-dev/quarto-cli/files/13886329/test.docx)

**Comment by cderv** [comment, marked as answer]
If you unnumber a section, it won't have number. That is why you get the 0.x for sub section. 

It seems to me that you want to change the style of your headings in docx. Using template could be a solution here (https://quarto.org/docs/output-formats/ms-word-templates.html#creating-templates) by changing the style of first level heading to have no number. If this is possible to do so in docx... I can't remember. 

Basically, each heading level has a different style in Word, so you can try to tweak it. 

Otherwise, built in `number-sections` does not allow you to have level 1 ignored and starts numbering at `1` directly. 

Unless I forgot something... 🤔
